firmware/library/L1_Drivers/system_clock.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+
+class SystemClock
+{
+ public:    
+      virtual void selectClockSource(bool option) = 0;
```

Comment:
What does it means to set option to true or false? I have no idea what this method actually does. Please name your variables in such a way that anyone reading this file could easily understand what happens when the parameters are set  to specific values.

Reply:
That goes for the rest of the methods. I can't seem to understand what they all do.